osbenchmark/workload_generator/corpus.py

```diff
@@ -62,24 +62,27 @@ def extract(client, output_path, index):
 
     logger = logging.getLogger(__name__)
 
-    total_docs = client.count(index=index)["count"]
-    if total_docs > 0:
-        logger.info("[%d] total docs in index [%s].", total_docs, index)
+    number_of_docs = client.count(index=index)["count"]
```

Comment:
I could perhaps have been clearer in my previous review, but leaving this variable as `total_docs` might be better since it is the total doc count.  On the other hand, the `number_of_docs_requested` and `ndocs_to_extract` variables are self-explanatory and appropriate.